sample/src/main/res/layout/activity_main.xml

```diff
@@ -82,5 +87,20 @@
         app:cit_drawableBottom="@drawable/ic_train_black_24px"
         app:cit_iconColor="#EF6C00"
         app:cit_iconHeight="@dimen/icon_size"
+        app:cit_iconWidth="@dimen/icon_size"
+        tools:ignore="HardcodedText" />
+
+    <com.github.aakira.compoundicontextview.CompoundIconTextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/margin_x_large"
+        android:drawablePadding="@dimen/margin_small"
+        android:gravity="center"
+        android:text="@string/right_to_left"
+        android:textColor="#795548"
+        android:textSize="@dimen/text_large"
+        app:cit_drawableStart="@drawable/ic_language_black_24px"
+        app:cit_iconColor="#4E342E"
```

Comment:
I used Brown 500 and 800 here from https://material.io/guidelines/style/color.html
because it reminds me something like Earth 🌍